### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by kirti-d**
the log files can be very long so we provide user with paginated view. 
tech used-html, js, java

### CODE REVIEW

**Comment by kirti-d** [comment]
@googlebot I fixed it.

File: error-analyzer/src/main/java/com/google/sps/data/ErrorsLogs.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package com.google.sps.data;
+
```
Comment (kirti-d) [inline error-analyzer/src/main/java/com/google/sps/data/ErrorsLogs.java:2]: @googlebot I fixed it.

File: error-analyzer/src/main/java/com/google/sps/servlets/Pagination.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+WebServlet("/pagination")
+public class Pagination extends HttpServlet {
+
+    private final int recordsPerPage = 3;
+    private final int noOfPages = 5;
+    private final int extraPageInFrontAndBack = 2;
+    private ArrayList < String > data = new ArrayList();
+
+    // java object need to be returned 
+    private class ReturnObj {
```
[the remark this answers is not in the document]
Comment (nishantnahata) [reply error-analyzer/src/main/java/com/google/sps/servlets/Pagination.java:54]: Why not return start page and end page numbers and let frontend decide which page to display?

File: error-analyzer/src/main/java/com/google/sps/servlets/Pagination.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
+// Copyright 2019 Google LLC
```
Comment (nishantnahata) [inline error-analyzer/src/main/java/com/google/sps/servlets/Pagination.java:1]: Please re-iterate this code to make it more readable, modular and try to remove repeated pieces.
For ex. 
Create a function: getDisplayData(fileType) which returns display data.

Also add JUnit test for this.

**Comment by nishantnahata** [comment]
Resolve conflicts. Also .war files are generated files? If yes, then add it to .gitignore

**Comment by kirti-d** [comment]
@googlebot I fixed it.

**Pull request closed without merging**